### PR TITLE
Check and cast presence channel user id to NSString

### LINKED ISSUE #175: PTPusherPresenceChannel memberRemoved has error
Hi,

I use the PTPusherPresenceChannel. And set the delegate for memberAdded and memberRemoved.
memberAdded works fine.

But when the memberRemoved trigger. the member parameter always nil.

I put the unsubscribe code like this

```
- (void)viewWillDisappear:(BOOL)animated
{
    [super viewWillDisappear:animated];
    
    [appDelegate.presenceChannel unsubscribe];
    [appDelegate.pusherClient disconnect];
    
    appDelegate.presenceChannel = nil;
    appDelegate.pusherClient = nil;
    
}
```

It has any wrong about leave the channel?

Thank you very much.

### PR DESCRIPTION
**Pull request by nero-tang**
Fix #175

Sometimes user_id can be a number and will yield nil when querying members.

Now user_id will be checked and cast to string before any process.

### CODE REVIEW

File: Pods/Pods.xcodeproj/xcuserdata/Nero.xcuserdatad/xcschemes/xcschememanagement.plist [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (hamchapman) [inline Pods/Pods.xcodeproj/xcuserdata/Nero.xcuserdatad/xcschemes/xcschememanagement.plist]: Do we want to include this file in the changes?

Comment (lukeredpath) [inline Pods/Pods.xcodeproj/xcuserdata/Nero.xcuserdatad/xcschemes/xcschememanagement.plist]: No.

Comment (hamchapman) [inline Pods/Pods.xcodeproj/xcuserdata/Nero.xcuserdatad/xcschemes/xcschememanagement.plist]: @nero-tang can you remove this file please?

Comment (nero-tang) [inline Pods/Pods.xcodeproj/xcuserdata/Nero.xcuserdatad/xcschemes/xcschememanagement.plist]: got it!

**Pull request merged**